conans/client/graph/graph.py

```diff
@@ -208,6 +208,15 @@ def __repr__(self):
         return repr(self.conanfile)
 
     def serialize(self):
+        def _topics():
```

Comment:
I remember changing ConanFile so topics would always be iterable. Did you get bitten by this?

Reply:
Good point! I just reviewed the code, and you're completely right! Thanks!